core/src/com/group66/game/cannon/BallManager.java

```diff
@@ -5,47 +5,102 @@
 
 import com.badlogic.gdx.Gdx;
 import com.badlogic.gdx.graphics.g2d.SpriteBatch;
-import com.group66.game.BustaMove;
+import com.group66.game.settings.Config;
 
+/**
+ * A Class to manage the Balls in the game.
+ */
 public class BallManager {
-	public static final int BALL_SPEED = 400;
-
-	Cannon cannon;
 	
-	/* Projectiles */
-	ArrayList<Ball> ballList = new ArrayList<Ball>();
-	ArrayList<Ball> ballDeadList = new ArrayList<Ball>();
-
-	/* Some balls to hit for testing */
-	ArrayList<Ball> ballStaticList = new ArrayList<Ball>();
-	ArrayList<Ball> ballStaticDeadList = new ArrayList<Ball>();
+	/** The cannon instance to shoot out. */
+	private Cannon cannon;
 	
-	private double deg_to_rad(double deg) {
-		return deg * Math.PI / 180.0;
-	}
+	/** The ball speed. */
+	private int ball_speed;
+	
+	/** The ball radius. */
+	private int ball_rad;
+	
+	/** The ball list. */
+	private ArrayList<Ball> ballList = new ArrayList<Ball>();
+	
+	/** The ball dead list. */
+	private ArrayList<Ball> ballDeadList = new ArrayList<Ball>();
```

Comment:
Its now just a constant, but in the future it should be made something smart. Its so the balls dont stay in memory when they are shot through the roof for now. When we implement the graph and other actual game features this should be adjusted or removed or whatever.